secure_download/.env.example

```diff
@@ -1,5 +1,3 @@
 # in test and production deployments set NODE_ENV accordingly
 NODE_ENV=docker
 JWKS_URI=http://127.0.0.1:5050/oauth/jwks
-
-UPLOAD_PATH_DATA_PRODUCTS=/path/to/uploaded/data_products
```

Comment:
The path where the uploaded file is to be placed on the Upload server is now provided by the core API. This enables the secure-upload API to be decoupled from business-entities like Raw Data, Data Products, etc.